onnxruntime/test/util/include/test_utils.h

```diff
@@ -24,5 +24,12 @@ void RunAndVerifyOutputsWithEP(const ORTCHAR_T* model_path,
                                const char* log_id,
                                std::unique_ptr<IExecutionProvider> execution_provider,
                                const NameMLValMap& feeds);
+
+// helper function that takes in model_data
+// used in nnapi qdq model tests
+void RunAndVerifyOutputsWithEPViaModelData(const std::string& model_data,
```

Comment:
nit, you can name this function as `RunAndVerifyOutputsWithEP` also, since both functions take different params, it's ok to have the same name